packages/menu/src/MenuGroup.ts

```diff
@@ -41,13 +46,38 @@ export class MenuGroup extends SpectrumElement {
         MenuGroup.instances += 1;
     }
 
+    private get menuRole() {
+        if (this.selects) {
+            return 'menu';
```

Comment:
I'm not sure this can ever see children through the slot in a screen reader, so the role here may actually go somewhere else. I'll have to visit the preview site to confirm.

Reply:
let me know if there's anything to do here.

Reply:
Thanks for reminding me, this looks good right now!

![image](https://user-images.githubusercontent.com/1156657/121033166-f0ceff80-c779-11eb-970b-0fcf93e69f1d.png)